### ISSUE TITLE
docker-compose

### ISSUE THREAD

**Issue by dunderrrrrr** [jhaals/yopass#349]
I'd like to start the docker containers using `docker-compose`.  

I'm currently running both memcached and yopass containers like this and it's working just fine. Perhaps this could be added to readme? Below is without TLS support. I'm running a separate nginx proxy for that.

```
version: '2.0'
services:
  memcached:
    restart: always
    image: "memcached"
    ports:
      - "11211:11211"
  yopass:
    restart: always
    image: "jhaals/yopass"
    ports:
      - "1337:1337"
    command: "--memcached=memcached:11211"
```
```
$ docker-compose up -d
```

Does the docker-compose file look okay? Any tips or trix @jhaals?

**Comment by jhaals** [comment]
This works for me 👍 feel free to submit a PR for adding it under /deploy

**Comment by jhaals** [comment]
Fixed by #350
